### PR TITLE
Update Install OpenCV-JavaScript tutorial page

### LINKED ISSUE #44: Update Introduction page in tutorials
We need to introduce more  of OpenCV-JavaScript and OpenCV-JavaScript Tutorials,  just like OpenCV-Python.

### PR DESCRIPTION
**Pull request by huningxin**
Fix #44

### CODE REVIEW

**Comment by huningxin** [comment]
@pancx , please take a look. Thanks!

**Comment by pancx** [comment]
looks good!

**Comment by huningxin** [comment]
Thanks for review!

File: doc/js_tutorials/js_setup/js_setup/js_setup.markdown [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	Launch the web browser to URL `http://localhost:8000/tests.html` which runs the unit tests automatically.
+
+    You can also run tests by Node.js
+	@code{.sh}
+	cd bin
+	npm install
+	node tests.js
+	@endcode
+
+	@node
```
Comment (pancx) [inline doc/js_tutorials/js_setup/js_setup/js_setup.markdown:113]: I think this node should be note

**Pull request merged**